File: src/main/resources/static/app/system/entities/ServerStatus.js
class ServerStatus{

	constructor(interfaceName, connected, details) {
		this.interfaceName = interfaceName;
		this.connected = connected;
		this.details = details;
	}
	
	getInterfaceName() {
		return this.interfaceName;
	}

	isConnected() {
		return this.connected;
	}
	
	getDetails() {
		return this.details;
	}
	


}

File: src/main/java/com/iotassistant/models/ServerStatus.java
package com.iotassistant.models;

public class ServerStatus {
	
	private String interfaceName;
	
	private boolean isConnected;
	
	private String detail;

	public ServerStatus(String interfaceName, boolean isConnected, String detail) {
		super();
		this.interfaceName = interfaceName;
		this.isConnected = isConnected;
		this.detail = detail;
	}

	public String getInterfaceName() {
		return interfaceName;
	}

	public boolean isConnected() {
		return isConnected;
	}

	public String getDetail() {
		return detail;
	}
	
	

}
